packages/core/src/agent/AgentModules.ts

```diff
@@ -145,6 +146,7 @@ function getDefaultAgentModules(agentConfig: AgentConfig) {
         maximumMediatorReconnectionIntervalMs: agentConfig.maximumMediatorReconnectionIntervalMs,
         mediatorPollingInterval: agentConfig.mediatorPollingInterval,
       }),
+    messagePickup: () => new MessagePickupModule({ maximumBatchSize: agentConfig.maximumMessagePickup }),
```

Comment:
the maximumMessagePickup is now removed from the agent config I think after my PR on agent config was merged